kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/types/listview/DataTypeListTest.java

```diff
@@ -857,75 +866,139 @@ public void testGetListElement() {
         assertEquals(expectedElement, actualElement);
     }
 
+    @Test
+    public void testImportDataObjects_NullOrEmpty() {
+        dataTypeList.importJavaClasses(null);
+        dataTypeList.importJavaClasses(Collections.emptyList());
+
+        verify(dataTypeList, never()).renameJavaClassToDMNName(any());
+        verify(dataTypeList, never()).insert(any());
+        verify(dataTypeList, never()).replace(any(), any());
+        verify(dataTypeList, never()).insertFields(any(), any());
+    }
+
     @Test
     public void testImportDataObjects() {
```

Comment:
This test method is too long in my opinion. Maybe you could refactor it a bit, by separating in different methods the initialization, the action, and the verification.
WDYT?

Reply:
@vpellegrino Sure thing.